examples/pvtol-nested-ss.py

```diff
@@ -36,102 +37,111 @@
 #
 
 # Design a simple lead controller for the system
-k = 200;  a = 2;  b = 50;
-Ci = k*tf([1, a], [1, b]);		# lead compensator
+k, a, b = 200, 2, 50
+Ci = k*tf([1, a], [1, b])  # lead compensator
 
 # Convert to statespace
-Ci = tf2ss(Ci);
+Ci = tf2ss(Ci)
 
 # Compute the loop transfer function for the inner loop
-Li = Pi*Ci;
+Li = Pi*Ci
 
 
 # Bode plot for the open loop process
-figure(1); 
-bode(Pi);
+plt.figure(1)
+bode(Pi)
 
 # Bode plot for the loop transfer function, with margins
-figure(2); 
-bode(Li);
+plt.figure(2)
+bode(Li)
 
 # Compute out the gain and phase margins
 #! Not implemented
 # (gm, pm, wcg, wcp) = margin(Li);
 
 # Compute the sensitivity and complementary sensitivity functions
-Si = feedback(1, Li);
-Ti = Li * Si;
+Si = feedback(1, Li)
+Ti = Li*Si
```

Comment:
Is there a guideline for when to use spaces around `*` versus not?  I see a mixture in this file.

Reply:
PEP8 talks about using space around operators:

> operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies). Use your own judgment; however, never use more than one space, and always have the same amount of whitespace on both sides of a binary operator.

Usually `*` and `/` have no space because they take higher priority over `+` and `-` but in a single statement I don't think this is relevant.  Personally I unless use spaces around `*` and `/` when they are between larger expressions (usually with parenthesis) so they don't appear joined together and to allow line-breaks.

I recommend removing space in this case (consistently throughout the file).

Reply:
Sounds reasonable.  I see some inconsistencies, but it looks like they are in different files.  As long as the spacing in a given example file is consistent, seems OK.

Reply:
I removed a bunch of space around `*` and `/` operators in various files.  Hopefully I caught them all.